src/porepy/models/slightly_compressible_flow_model.py

```diff
@@ -109,63 +95,27 @@ def _assign_discretizations(self) -> None:
         """
 
         super()._assign_discretizations()
-        gb = self.gb
-        grid_list = [g for g, _ in gb.nodes()]
-
-
-        # Ad representation of discretizations
-        mass_mat = pp.MassMatrix(self.parameter_key)
-        mass_mat.discretize(gb)
-
-        # Ad variables
-        p = self._eq_manager.merge_variables([(g, self.variable) for g in grid_list])
-
-        # Update equations
-        subdomain_flow_eq = self._eq_manager.equations["subdomain_flow"]
-        interface_flow_eq = self._eq_manager.equations["interface_flow"]
-
-        mass = (
-            mass_mat * (p - p.previous_timestep()) / self.time_step
+        
+        if not hasattr(self, "dof_manager"):
```

Comment:
I think this is not necessary, since the super call to the parent class method asserts this attribute is available

Reply:
Sure this is not necessary.